### PR TITLE
Devlog

### PR DESCRIPTION
**Pull request by SchaeferAdam**

### CODE REVIEW

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/ForgotPassword.cshtml [hunk 1]
```diff
@@ -4,23 +4,44 @@
...
-<div class="row">
-    <div class="col-md-4">
-        <form method="post">
-            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
-            <div class="form-floating">
-                <input asp-for="Input.Email" class="form-control" autocomplete="username" aria-required="true" />
-                <label asp-for="Input.Email" class="form-label"></label>
-                <span asp-validation-for="Input.Email" class="text-danger"></span>
+
+<head>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/ForgotPassword.cshtml:8]: remove <head> section, it's already in _layout

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/ForgotPassword.cshtml [hunk 2]
```diff
@@ -4,23 +4,44 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&display=swap" rel="stylesheet">
+    <link href="~/css/StyleSheet.css" rel="stylesheet" />
+</head>
+
+<body>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/ForgotPassword.cshtml:15]: remove lines with `<body>` and `</body>` - body is already in _layout

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/Login.cshtml [hunk 3]
```diff
@@ -5,81 +5,69 @@
...
-            </form>
-        </section>
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&display=swap" rel="stylesheet">
+    <link href="~/css/StyleSheet.css" rel="stylesheet" />
+</head>
+
+<body>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/Login.cshtml:15]: and `<body>`, `</body>`

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/Register.cshtml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+﻿@page
+@model RegisterModel
+@{
+    ViewData["Title"] = "Register";
+}
+
+<head>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/Register.cshtml:7]: like above

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/Register.cshtml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&display=swap" rel="stylesheet">
+    <link href="~/css/StyleSheet.css" rel="stylesheet" />
+</head>
+
+<body>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/Register.cshtml:14]: like previous

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/ResendEmailConfirmation.cshtml [hunk 6]
```diff
@@ -0,0 +1,45 @@
+﻿@page
+@model ResendEmailConfirmationModel
+@{
+    ViewData["Title"] = "Resend email confirmation";
+}
+
+<head>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/ResendEmailConfirmation.cshtml:7]: you now know what to do here and with body

File: BloodTypeC.WebApp/Areas/Identity/Pages/Account/ResetPassword.cshtml [hunk 7]
```diff
@@ -4,34 +4,49 @@
...
-            <div class="form-floating">
-                <input asp-for="Input.Password" class="form-control" autocomplete="new-password" aria-required="true" />
-                <label asp-for="Input.Password" class="form-label"></label>
-                <span asp-validation-for="Input.Password" class="text-danger"></span>
-            </div>
-            <div class="form-floating">
-                <input asp-for="Input.ConfirmPassword" class="form-control" autocomplete="new-password" aria-required="true" />
-                <label asp-for="Input.ConfirmPassword" class="form-label"></label>
-                <span asp-validation-for="Input.ConfirmPassword" class="text-danger"></span>
+<head>
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Areas/Identity/Pages/Account/ResetPassword.cshtml:7]: head and body again

File: BloodTypeC.WebApp/Program.cs [hunk 8]
```diff
@@ -16,6 +17,7 @@ public class Program
         public static void Main(string[] args)
         {
             var builder = WebApplication.CreateBuilder(args);
+                        var connectionString = builder.Configuration.GetConnectionString("BloodTypeCWebAppContextConnection") ?? throw new InvalidOperationException("Connection string 'BloodTypeCWebAppContextConnection' not found.");
```
Comment (miniatureluck) [inline BloodTypeC.WebApp/Program.cs:20]: remove this line

File: BloodTypeC.WebApp/Views/FavoriteBeers/Favorites.cshtml
Comment (miniatureluck) [inline BloodTypeC.WebApp/Views/FavoriteBeers/Favorites.cshtml:1]: can't
`<div class="button-margin">
    <div class="buttons">
        <div class="form-btn">`
be just one div? that applies to all views, btw : )

File: BloodTypeC.WebApp/appsettings.json
Comment (miniatureluck) [inline BloodTypeC.WebApp/appsettings.json:1]: this shouldn't be added, we already have the string in our own context

**Review by miniatureluck — CHANGES_REQUESTED** [review]

File: BloodTypeC.WebApp/Views/FavoriteBeers/Favorites.cshtml
Comment (SchaeferAdam) [reply BloodTypeC.WebApp/Views/FavoriteBeers/Favorites.cshtml:1]: number of divs and style classses depends on how scaffolding works and what is used. somtimes you need to add extra space or positioning in vertical orientation thats why there are different type of button classes used when needed

File: BloodTypeC.WebApp/Views/Beer/Create.cshtml
Comment (miniatureluck) [inline BloodTypeC.WebApp/Views/Beer/Create.cshtml:1]: it looks like you are tring to revert the create beer view to the old version without new functionality that i've added, please do not do that and keep the one from dev branch

File: BloodTypeC.WebApp/Views/Beer/Details.cshtml
Comment (miniatureluck) [inline BloodTypeC.WebApp/Views/Beer/Details.cshtml:1]: again, you are trying to revert the changes that i've made to this view

File: BloodTypeC.WebApp/appsettings.Development.json
Comment (miniatureluck) [inline BloodTypeC.WebApp/appsettings.Development.json:1]: why did you remove this?

File: BloodTypeC.WebApp/appsettings.json
Comment (miniatureluck) [inline BloodTypeC.WebApp/appsettings.json:1]: why did you remove this?

**Review by miniatureluck — CHANGES_REQUESTED** [review]
